Sources/ShopifyCheckoutSheetKit/CheckoutDelegate.swift

```diff
@@ -27,7 +27,7 @@ import UIKit
 /// A delegate protocol for managing checkout lifecycle events.
 public protocol CheckoutDelegate: AnyObject {
 	/// Tells the delegate that the checkout successfully completed.
-	func checkoutDidComplete()
+	func checkoutDidComplete(event: CheckoutCompletedEvent)
```

Comment:
Since we're updating the protocol here, this will be a breaking change and will require a major version bump. 

We could avoid a major bump by deprecating this method and creating a new one with the event arg:

```swift
@deprecated
func  checkoutDidComplete()

func checkoutDidCompleteWithEvent(event: CheckoutCompletedEvent)
```

@kiftio we don't have the option to overload here in Swift. Is that what you did in the Android repo?